website/pages/blog/configuring-cq-with-localstack.md

```diff
@@ -0,0 +1,84 @@
+---
+title: CloudQuery and LocalStack
+tag: integration
+date: 2022/12/23
```

Comment:
Considering this is not merged we need a better date here :)

Reply:
How to guides should not have a date, so we should remove it (we wanted them sorted by title only)

Reply:
I agree... I have made [a new issue](https://github.com/cloudquery/cloudquery/issues/6019) to track this work. In the meantime I have updated the date